src/status_im2/contexts/quo_preview/transaction_sheet/transaction_sheet.cljs

```diff
@@ -0,0 +1,91 @@
+(ns status-im2.contexts.quo-preview.transaction-sheet.transaction-sheet
+  (:require [quo2.components.tabs.account-selector :as acc-sel]
+            [utils.i18n :as i18n]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [status-im2.contexts.quo-preview.tabs.segmented-tab]
+            [quo2.components.tabs.segmented-tab :as quo2]
+            [status-im2.contexts.shell.jump-to.utils :as utils]
+            [quo2.components.inputs.locked-input.view :as locked-input]
+            [quo2.components.markdown.text :as text]))
+
+(let [{:keys [width]} (utils/dimensions)]
+  (def screen-width width))
+(def account-selector-width (* 0.895 screen-width))
+(def shared-selector-list-data
+  {:show-label?  false
+   :transparent? false
+   :style        {:width  account-selector-width
+                  :height 40}})
+(def unique-selector-list-data
+  [{:account-text  "Drakaris account"
+    :account-emoji "🔥"}
+   {:account-text  "Daenerys account"
+    :account-emoji "👸"}])
+(def selector-list-data
+  (map #(merge % shared-selector-list-data)
+       unique-selector-list-data))
+
+(def label-style
+  {:color          colors/neutral-50
+   :letter-spacing -0.039
```

Comment:
the quo2 components - `quo/text` already has all of these styles encapsulated in them. you can use that instead and set with keywords. there are many examples in the codebase, or feel to reach out off github and I can send you some examples 👍